Cargo.toml

```diff
@@ -16,55 +16,50 @@ edition = "2018"
 [dependencies]
 bit-vec = "0.6"
 blake2s_simd = "0.5"
-ff = { version = "0.3.0", package = "fff" }
-groupy = "0.4.1"
-rand_core = "0.5"
+ff = "0.11.0"
+group = "0.11.0"
+rand_core = "0.6"
 byteorder = "1"
 log = "0.4.8"
 lazy_static = "1.4.0"
-rand = "0.7"
+rand = "0.8"
 rayon = "1.5.0"
 memmap = "0.7.0"
 thiserror = "1.0.10"
 rustc-hash = "1.1.0"
 num_cpus = "1"
 crossbeam-channel = "0.5.0"
 digest = "0.9.0"
-itertools = "0.9.0"
+itertools = "0.10.0"
 bincode = "1.3.1"
 serde = { version = "1.0", features = ["derive"] }
 sha2 = "0.9"
-
-# blst feature
-blstrs = { version = "0.3", optional = true }
-
-# pairing feature
-paired = { version = "0.22.0", optional = true }
+blstrs = { git = "https://github.com/filecoin-project/blstrs", branch = "master" }
+pairing = "0.21"
+yastl = "0.1.2"
 
 # gpu feature 
 rust-gpu-tools = { version = "0.4.0", optional = true }
-ff-cl-gen = { version = "0.3.0", optional = true }
+ec-gpu = { git = "https://github.com/filecoin-project/ec-gpu", branch = "master", optional = true }
+ec-gpu-gen = { git = "https://github.com/filecoin-project/ec-gpu", branch = "master", optional = true }
```

Comment:
Just to clarify, is ff-cl-gen going to be released before this is merged, or are we sitting at `master` while things are in flux?

Reply:
The plan is to sit on `master` until at least the cuda things are also merged